frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetailsView.tsx

```diff
@@ -155,21 +141,9 @@ const ModelVersionDetailsView: React.FC<ModelVersionDetailsViewProps> = ({
           >
             <InlineTruncatedClipboardCopy testId="model-version-id" textToCopy={mv.id} />
           </DashboardDescriptionListGroup>
-          {Object.values(pipelineCustomProperties).every((value) => !!value) && (
+          {modelArtifact && (
```

Comment:
We need to not render the DashboardDescriptionListGroup unless we are fully rendering one of the links inside it - with this logic, if the artifact exists but does not have the model source properties we'll get a "Registered from" title with empty contents.

Let's just render the component itself and then conditionally use `DashboardDescriptionListGroup` inside it.

```tsx
{modelArtifact && (
  <ModelVersionRegisteredFromLink modelArtifact={modelArtifact} />
)}
```